client.go

```diff
@@ -618,22 +618,23 @@ func (c *Client) GetReadyMessagesCount(queueName string) (msgCount int, err erro
 	return
 }
 
-// ConsumeReadyMessages fetches given number of messages from queue
-func (c *Client) ConsumeReadyMessages(queueName string, numberOfMessages int) (msgs []amqp.Delivery, err error) {
+// GetReadyMessages fetches given number of messages from queue
```

Comment:
it is not true: we will return `at most` given number of messages

Reply:
Shall i update it with `GetReadyMessages fetches at most given number of messages from queue` ?

Reply:
yes